apps/test/unit/lib/ui/SchoolInfoInterstitialTest.jsx

```diff
@@ -156,6 +157,43 @@ describe('SchoolInfoInterstitial', () => {
     );
   });
 
+  it('calls onChange function when country is changed from US to non-US', () => {
+    let wrapper = mount(
+      <SchoolInfoInterstitial
+        {...MINIMUM_PROPS}
+        scriptData={{
+          ...MINIMUM_PROPS.scriptData,
+          existingSchoolInfo: {
+            school_id: '123',
+            country: 'United States',
+            school_type: 'public',
+            school_name: 'Test School',
+            full_address: 'Seattle'
+          }
+        }}
+      />
+    );
+
+    expect(wrapper).to.containMatchingElement(
+      <SchoolInfoInputs
+        country={'United States'}
+        schoolType={'public'}
+        ncesSchoolId={'123'}
+        schoolName={'Test School'}
+        schoolLocation={'Seattle'}
+        useGoogleLocationSearch={true}
+        onCountryChange={wrapper.instance().onCountryChange}
+        onSchoolTypeChange={wrapper.instance().onSchoolTypeChange}
+        onSchoolChange={wrapper.instance().onSchoolChange}
+        onSchoolNotFoundChange={wrapper.instance().onSchoolNotFoundChange}
+      />
+    );
+    const event = {value: 'Sweden'};
+    wrapper.instance().onCountryChange(undefined, event);
+    expect(wrapper.state('country')).to.equal('Sweden');
+    expect(wrapper.state('ncesSchoolId')).to.equal('');
```

Comment:
I'd love to see something like

```js
    expect(wrapper.state('country')).to.equal('United States');
    expect(wrapper.state('ncesSchoolId')).to.equal('123');
    wrapper.instance().onCountryChange(undefined, {value: 'Sweden'});
    expect(wrapper.state('country')).to.equal('Sweden');
    expect(wrapper.state('ncesSchoolId')).to.equal('');
```

So we can confirm the values actually _changed_